requirements-ext.txt

```diff
@@ -1 +1,2 @@
 numpy>=1.6.1
+singledispatch
```

Comment:
Yes, but it now came up as missing in Travis CI, since for the first time tests are added.